src/frontend/screens/YourTeam/CollaboratorInfo.tsx

```diff
@@ -123,7 +123,7 @@ export const CollaboratorInfo: NativeNavigationComponent<
               navigation.navigate('LeaveProject');
             }}
           />
-        ) : ownRoleIsCoordinator ? (
+        ) : ownRoleIsCoordinator && name ? (
```

Comment:
the frontend always validates so there always is a name. But in the slight chance that something happens and there is no name, I don't think we should block the user. So instead of doing this check we can just pass an empty string:

```ts

onPress={() => {
              navigation.replace('RemoveDevice', {
                deviceId: route.params.deviceId,
                deviceName: name || "",
              });
            }}
```